dev/scf/docs/accessing.md

```diff
@@ -0,0 +1,18 @@
+# Accessing the SCF cluster
+
+## When using NGINX Ingress Controller
+
+NGINX Ingress Controller with Minikube enables the access from the host to the SCF cluster. Follow the
```

Comment:
Are there similar changed instructions for us using `KinD` ?
I later see `(optional) NGINX Ingress Controller for external access - ONLY FOR MINIKUBE` so I guess I cannot test the nginx ingress changes via KinD ?

Or will nginx ingress work generally and it is only the external access I cannot have ? (Although that would block me from using `cf`, etc. to actually talk to the CF deployment, right ?

Reply:
Right. The purpose of the ingress controller is to give external access to whatever backends it exposes. This is why I marked as an option only for Minikube. With KinD we cannot expose anything except using `kubectl port-forward`, which doesn't help us here.